src/main/java/me/hugeblank/allium/lua/api/JavaLib.java

```diff
@@ -0,0 +1,96 @@
+package me.hugeblank.allium.lua.api;
+
+import me.hugeblank.allium.lua.type.UserdataFactory;
+import org.squiddev.cobalt.Constants;
+import org.squiddev.cobalt.LuaError;
+import org.squiddev.cobalt.LuaState;
+import org.squiddev.cobalt.Varargs;
+import org.squiddev.cobalt.lib.LuaLibrary;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JavaLib {
+    public static LuaLibrary create() {
+        return LibBuilder.create("java")
+                .add("create", JavaLib::createInstance)
+                .add("getRawClass", JavaLib::getClassObject)
+                .add("cast", JavaLib::cast)
+                .build();
+    }
+
+    private static Varargs createInstance(LuaState state, Varargs args) throws LuaError {
+        Class<?> clazz;
+        try {
+            clazz = Class.forName(args.arg(1).checkString());
```

Comment:
Call me crazy but I don't think this would work in production due to the mapping issue

Reply:
This happens in the other two methods also

Reply:
Yeah these are temp. They will get mappings support later (should be relatively simple)

Reply:
It is, `Allium.MAPPINGS` has them. Tragically though you have to iterate over the values. I believe I have an example somewhere in `UserdataFactory`.

Reply:
I'm sure iteration could be skipped vy using 2 maps with reversed key-value

Reply:
That's probably true and something I'll consider

Reply:
> I'm sure iteration could be skipped vy using 2 maps with reversed key-value

Or just a Guava BiMap

Reply:
> > I'm sure iteration could be skipped vy using 2 maps with reversed key-value
> 
> Or just a Guava BiMap

I replaced it with custom class for easier usage

Reply:
Absolutely beautiful.